raiden_contracts/contracts/TokenNetworkRegistry.sol

```diff
@@ -39,13 +47,16 @@ contract TokenNetworkRegistry is Utils {
         chain_id = _chain_id;
         settlement_timeout_min = _settlement_timeout_min;
         settlement_timeout_max = _settlement_timeout_max;
+
+        administrator = msg.sender;
     }
 
     /// @notice Deploy a new TokenNetwork contract for the Token deployed at
     /// `_token_address`.
     /// @param _token_address Ethereum address of an already deployed token, to
     /// be used in the new TokenNetwork contract.
     function createERC20TokenNetwork(address _token_address)
+        onlyAdmin
```

Comment:
I had a different idea about how this should be done.
The idea would be to allow only one token network to exist instead of allowing only the administrator to deploy one.

The end result would indeed be the same but would at least agree with our limitations and give less power to the administrator.